app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -836,7 +845,8 @@ public function __construct(
         MediaGalleryProcessor $mediaProcessor = null,
         StockItemImporterInterface $stockItemImporter = null,
         DateTimeFactory $dateTimeFactory = null,
-        ProductRepositoryInterface $productRepository = null
+        ProductRepositoryInterface $productRepository = null,
+        LinkProcessor $linkProcessor = null
```

Comment:
@akaplya can this optional constructor dependency be included in a patch release?

Reply:
I think it's a not a B/C breaker